### DISCUSSION TITLE
How to convert tibble to CSV.

### DISCUSSION THREAD

**Discussion by Aaron116666** [jamesjiadazhan/dietaryindex#219] [Q&A] [answered]
When I run the code，as follow “DII_NHANES_FPED(FPED_PATH2=FPED_PATH_2, NUTRIENT_PATH2=NUTRIENT_PATH_2, DEMO_PATH=DEMO_PATH)”. How to convert tibble to CSV?
![132](https://github.com/jamesjiadazhan/dietaryindex/assets/148944903/16f73cf8-c329-4229-b5d7-4e35030f2a85)

**Comment by jamesjiadazhan** [comment, marked as answer]
You just need to store the DII result in a new variable and then use write_csv to save the result to your computer. Like below:

```
DII_result = DII_NHANES_FPED(FPED_PATH2=FPED_PATH_2, NUTRIENT_PATH2=NUTRIENT_PATH_2, DEMO_PATH=DEMO_PATH)
write_csv(DII_result, "DII_result.csv")
```

Let me know if you have other questions. If possible, please give our work a Star on GitHub and cite our work when you publish the work. It helps recognize and appreciate our intellectual efforts and contributions.
![image](https://github.com/jamesjiadazhan/dietaryindex/assets/108076575/fb3d6b28-ccbc-4991-a8d3-139cf2096cd4)

Jiada James Zhan, Rebecca A Hodge, Anne Dunlop, et al. Dietaryindex: A User-Friendly and Versatile R Package for Standardizing Dietary Pattern Analysis in Epidemiological and Clinical Studies. bioRxiv. Published online August 07, 2023:2023.08.07.548466. doi:10.1101/2023.08.07.548466

**Comment by Aaron116666** [comment]
Thank you very much for your answer, I have solved this problem. 
Currently, when I run the following code with "MEDI", "AHEI" and "DASH", I encounter the same problem as follows “Error in DASH_NHANES_FPED(FPED_PATH = FPED_PATH_1, NUTRIENT_PATH = NUTRIENT_PATH_1,  : 
  unused arguments (FPED_PATH = FPED_PATH_1, NUTRIENT_PATH = NUTRIENT_PATH_1, DEMO_PATH = DEMO_PATH)”. 
How can I solve it?
![1](https://github.com/jamesjiadazhan/dietaryindex/assets/148944903/b174f125-26b2-4de6-a0fe-7f79a258add4)

**Reply by jamesjiadazhan** [reply]
Please check the manual for additional information. You are not using the functions appropriately. Not all the functions have the same arguments to be used.

https://jamesjiadazhan.github.io/dietaryindex_manual/
